pkg/agent/openflow/pipeline.go

```diff
@@ -702,7 +702,7 @@ func (c *client) localProbeFlow(localGatewayIP net.IP, category cookie.Category)
 		Done()
 }
 
-func (c *client) snatFlows(uplinkOfport uint32, bridgeLocalPort int, nodeIP net.IP, category cookie.Category) []binding.Flow {
+func (c *client) snatFlows(uplinkOfport uint32, bridgeLocalPort int, nodeIP net.IP, localSubnet net.IPNet, category cookie.Category) []binding.Flow {
```

Comment:
Feel snatFlows does not match the functions of this func. Should we rename it to sth. like bridgeAndUplinkFlows?

Reply:
Sure, I will update.